spec/controllers/generic_works_controller_spec.rb

```diff
@@ -136,12 +134,8 @@
                                  file_set: {}
           }.to change(FileSet, :count).by(2)
           created_files = FileSet.all
-          ["https://dl.dropbox.com/fake/blah-blah.Getting%20Started.pdf", "https://dl.dropbox.com/fake/blah-blah.filepicker-demo.txt.txt"].each do |url|
-            expect(created_files.map(&:import_url)).to include(url)
-          end
-          ["filepicker-demo.txt.txt", "Getting+Started.pdf"].each do |filename|
-            expect(created_files.map(&:label)).to include(filename)
-          end
+          expect(created_files.map(&:import_url)).to include(url1, url2)
```

Comment:
You **already** have to understand created_files to understand this test file and these tests.  This is more DRY and utilizes the expressivity of `include`.   Loops over `expect` are anti-pattern in rspec, especially when include can already check for multiple values.